compatibility-certification-request.md

```diff
@@ -0,0 +1,20 @@
+- [ ] Statement of Acceptance of the terms of the EFTL
+  // Add here
+- [ ] Product Name, Version and download URL (if applicable)
+  // Add here
+- [ ] Specification Name, Version and download URL
+  // Add here
+- [ ] TCK Version, digital SHA-256l fingerprint and download URL
+  // Add here
+- [ ] Implementation runtime Version(s) tested
+  // Add here
+- [ ] Public URL of TCK Results Summary
+  // Add here
+- [ ] Any Additional Specification Certification Requirements
+  // Add here
+- [ ] Java runtime used to run the implementation
+  // Add here
+- [ ] Summary of the information for the certification environment, operating system, cloud, ...
+  // Add here
+- [ ] A statement attesting that all TCK requirements have been met, including any compatibility rules
+  // Add here
```

Comment:
As above, how about:

- [ ] By checking this box I attest that all TCK requirements have been met, including any compatibility rules.